bin/graph_altimeter_batch.py

```diff
@@ -51,8 +51,13 @@ def run_scan():
     else:
         accounts = get_aws_accounts(asset_inventory_api_url)
 
-    for account_id in accounts:
-        logger.info("scanning account %s", account_id)
+    for i, account_id in enumerate(accounts):
+        logger.info(
+            "scanning account %s (%s/%s)",
```

Comment:
Given that the account index and the number of accounts are integers, isn't it better to use `%d`?

```suggestion
            "scanning account %s (%d/%d)",
```

Reply:
Yep okey.